### DISCUSSION TITLE
Mint several NFTs at once

### DISCUSSION THREAD

**Discussion by lefeverela** [metaplex-foundation/metaplex#2079] [Q&A]
Hello,

I am trying to play around with the code to be able to mint several NFT at once. Like for example I want my users to be able to mint 10 NFT at once in a single transaction.

I was playing around with the candy-machine.ts and this block of code., and replace the 1 by 2 or other numbers.
    Token.createMintToInstruction(
      TOKEN_PROGRAM_ID,
      mint.publicKey,
      userTokenAccountAddress,
      payer,
      [],
      1,
    ),

But the phantom wallet does not seems to like it and show me the following message ahead of minting: "Transaction may fail to confirm".

Any idea how to change the code to be able to mint several NFT at once? Or what am I doing wrong?

Thanks

**Comment by spreadcharm** [comment]
That's impossible, I think.
It's related to how javascript interacts with Solana program.
Although you tried and tried, since you should interact Candymachine program, it cannot be done in one interaction.
This means you need multiple sign action from phantom wallet.
If you want do really, you should update phantom wallet too, not only candymachine ui.

**Reply by bornnel** [reply]
what does this mean in the .env code "SPL TOKEN MINT:" It has addresses

**Comment by cesar4design** [comment]
I have found this website, where you can mount several nfts at the same time, and use a candy machine under the front end. Any help on how to do it?

https://lovebears.xyz/
